src/ServerlessOpenApiDocumentation.ts

```diff
@@ -1,4 +1,4 @@
-import * as c from 'chalk';
+import chalk from 'chalk';
```

Comment:
looks like more than just updating from yarn to npm?

Reply:
This broke linting... somehow

Reply:
ah ok, fair enough. weird!